gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/java/io/gravitee/rest/api/portal/rest/resource/auth/ConsoleAuthenticationResource.java

```diff
@@ -89,7 +91,14 @@ public Response redirectTo(
             // connect the user
             super.connectUser(jwt.getSubject(), httpResponse);
             String url = installationAccessQueryService.getPortalUrl(environmentId);
-            var versionQueryParam = version != null && version.equals(PORTAL_NEXT_VERSION) ? PORTAL_NEXT_PATH : "";
+            var versionQueryParam = ""; // Default to no Version path
+            if (version != null) {
+                if (version.equals(PORTAL_NEXT_VERSION)) {
+                    versionQueryParam = PORTAL_NEXT_PATH;
+                } else if (version.equals(PORTAL_CLASSIC_VERSION)) {
+                    versionQueryParam = PORTAL_CLASSIC_PATH;
+                }
+            }
```

Comment:
Same suggestion here

```suggestion
            var versionQueryParam = switch(vesion) {
                case PORTAL_NEXT_VERSION -> PORTAL_NEXT_PATH;
                case PORTAL_CLASSIC_VERSION -> PORTAL_CLASSIC_PATH;
                case null, default -> "";
            };
```